include/PathUtil.h

```diff
@@ -0,0 +1,29 @@
+#ifndef PATHUTIL_H
+#define PATHUTIL_H
+
+#include "lmms_export.h"
+
+#include <QDir>
+
+namespace PathUtil
+{
+	enum Base { AbsoluteBase, ProjectDirBase, FactorySampleBase, UserSampleBase, UserVSTBase, PresetBase,
+		UserLADSPABase, DefaultLADSPABase, UserSoundfontBase, DefaultSoundfontBase, UserGIGBase, DefaultGIGBase };
```

Comment:
Consider using an `enum class`, then removing the `Base` suffix from all of these. You can then refer to them as, e.g., `Base::Absolute`. This has the advantage of stronger typing (no implicit conversions to/from `int`), and doesn't fill the namespace with constants without any context.